client/src/components/Search/index.js

```diff
@@ -0,0 +1,151 @@
+    ));
+
+  onSubmit = () => {};
+
+  setKeyword = value => {
+    this.setState({ keyword: value, viewKeywords: false });
+  };
+
+  hideKeyword = () => {
+    this.setState(({ viewKeywords }) => ({ viewKeywords: !viewKeywords }));
+  };
+
+  dropdownFilter = (input, option) =>
+    option.props.children.toLowerCase().indexOf(input.toLowerCase()) >= 0;
+
+  render() {
+    const { country, city, keyword, viewKeywords, lookingFor } = this.state;
+
+    return (
+      <form className="search__form">
+        <div className="country-city-container">
+          <div className="autocomplete-box">
+            <div>WHICH COUNTRY?</div>
+            <Select
+              showSearch
+              placeholder="Select"
+              optionFilterProp="children"
+              onChange={value => this.setState({ country: value, city: '' })}
+              value={country}
+              filterOption={this.dropdownFilter}
+            >
+              {this.renderOptions(world.getCountryNames())}
+            </Select>
+          </div>
+          <div className="autocomplete-box">
+            <div>WHICH CITY?</div>
+            <Select
+              showSearch
+              placeholder="Select"
+              optionFilterProp="children"
+              onChange={value => {
+                this.setState({ city: value });
+              }}
+              value={city}
+              filterOption={this.dropdownFilter}
+            >
+              {country &&
+                world.getCities(country) !== undefined &&
+                this.renderOptions(world.getCities(country))}
+            </Select>
+          </div>
+        </div>
+
+        <div className="type-filter__container">
+          <Radio.Group
+            className="radio-group"
+            value={lookingFor}
+            buttonStyle="solid"
+          >
+            <p className="button-label">What Are You Looking For</p>
+            <Radio.Button
+              className="radio-button"
+              onClick={() =>
+                this.setState({ lookingFor: this.lookingFor.STAY })
+              }
+            >
+              Stay
+            </Radio.Button>
+            <Radio.Button
+              className="radio-button"
+              onClick={() => this.setState({ lookingFor: this.lookingFor.EAT })}
+            >
+              Eat & drink
+            </Radio.Button>
+            <Radio.Button
+              className="radio-button"
+              onClick={() =>
+                this.setState({ lookingFor: this.lookingFor.SHOP })
+              }
+            >
+              Shop
+            </Radio.Button>
+          </Radio.Group>
+          <div className="type-filter__container__filter">
+            <p className="button-label">Filter</p>
+            <Button
+              className="keywords-btn"
+              onClick={() => {
+                this.setState(state => ({
+                  viewKeywords: !state.viewKeywords,
+                }));
+              }}
```

Comment:
declaration this function and use it 
`this.toggleKeywordList`
it seems like hideKeyword function you declaration  above, if yes rename it to toggleKeywordList